File: Carlene_Lab6_4/app/src/main/java/com/example/carlene_lab6_4/MainActivity.java
package com.example.carlene_lab6_4;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.GridLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import java.util.ArrayList;
import java.util.List;

public class MainActivity extends AppCompatActivity {
    RecyclerView recyclerView;
    List<MyModel> myModelList;
    CustomAdapter customAdapter;

    @Override
    protected void onCreate(Bundle savedInstanceState){
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        displayItems();
    }

    private void displayItems() {
        recyclerView = findViewById(R.id.recycler_main);
        recyclerView.setHasFixedSize(true);
        recyclerView.setLayoutManager(new GridLayoutManager(this, 1));
        myModelList = new ArrayList<>();
        myModelList.add(new MyModel("Carlene", 20));
        myModelList.add(new MyModel("Sidney", 19));
        myModelList.add(new MyModel("Janice", 17));
        myModelList.add(new MyModel("Maxy", 2));
        myModelList.add(new MyModel("Ace", 3));
        customAdapter = new CustomAdapter(this, myModelList);
        recyclerView.setAdapter(customAdapter);
    }

}

File: Carlene_Lab6_3/app/src/main/java/com/example/carlene_lab6_3/MainActivity.kt
package com.example.carlene_lab6_3

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.databinding.DataBindingUtil
import com.example.carlene_lab6_3.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = DataBindingUtil.setContentView(this,R.layout.activity_main)

        val user = User ("Robin", "Singla", "27")
        binding.user = user


    }
}